src/layouts/Header.js

```diff
@@ -1,14 +1,92 @@
 import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { AppBar, Toolbar, Typography, IconButton, Switch, FormControlLabel, FormGroup, MenuItem, Menu  } from '@material-ui/core';
+import MenuIcon from '@material-ui/icons/Menu';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import Link from '@material-ui/core/Link';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  menuButton: {
+    marginRight: theme.spacing(2),
+  },
+  title: {
+    flexGrow: 1,
+  },
+}));
+
 /**
  * For the header
  * @param {} props 
  */
-function Header(props) {
+
+export default function Header() {
+  const classes = useStyles();
```

Comment:
Also this uses hooks while our project mainly uses class-based components? Should we switch to functional based components and hooks for consistency?